pkg/apis/pipeline/v1/resultref_test.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 func TestNewResultReference(t *testing.T) {
+	idx := 1
```

Comment:
Does this really covers the change? IIUC we need a test case that references an empty array with [*] and test the `ResultIndex` is nil?

Reply:
Oh thanks, I think I forgot to add a test case here